api/nodes/serializers.py

```diff
@@ -243,10 +232,18 @@ def create(self, validated_data):
         return contributor
 
 
+class NodeContributorsCreateSerializer(NodeContributorsSerializer):
+    """
+    Overrides NodeContributorsSerializer to add target_type field
+    """
+    target_type = TargetTypeField(target_type='users')
```

Comment:
The parent serializer defines ` create` method, but as far as I can tell that is only used by views that rely on this (child) serializer. (for POST requests)

Can we move the `NodeContributorsSerializer.create` method to this class, where the logic is more relevant?